code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -24,6 +24,7 @@
 	var/list/embedded_objects = list()
 	var/held_index = 0 //are we a hand? if so, which one!
 	var/is_pseudopart = FALSE //For limbs that don't really exist, eg chainsaws
+	var/render_like_organic = FALSE // TRUE is for when you want a BODYPART_ROBOTIC to pretend to be a BODYPART_ORGANIC.
```

Comment:
just make a BODYPART_IPC or BODYPART_SYNTH

instead of this var hackery

Reply:
If I add a new BODYPART_IPC, it'll probably fix IPCs being spaceworthy and coldproof/heatproof as well, so will do.